src/main/java/org/example/model/UrlPattern.java

```diff
@@ -0,0 +1,25 @@
+package org.example.model;
+
+public enum UrlPattern {
+    HTML("/html"), JSON("/json"), HELLO("/hello");
+
+    private final String url;
+
+    UrlPattern(String url) {
+        this.url = url;
+    }
+
+    public String getUrl() {
+        return "/DeveloperApi" + url;
+    }
+
+    // Метод для отримання UrlPattern за рядком URL
```

Comment:
What if some developer from England, Germany or India will check your code...
He will not understand this text...

Reply:
fix